kubeflow-metadata-envoy-config.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: kubeflow-metadata-envoy-config
```

Comment:
Just to double check, if this is related to another package in wolfi could we instead add this config as a subpackage of that other wolfi package?  I.e. rather than having a new top level package?

Reply:
We could move it to a subpackage, yes. What did you have in mind?

Reply:
Moved it into the main kubeflow as a subpackage.